buildcomponents.yaml

```diff
@@ -0,0 +1,13 @@
+.migrate:
```

Comment:
This should also have
```yaml
except:
    - schedules
```

Reply:
I'm not sure about that... its always used in another task via extension.  Do we want to force the build job to have except schedules? Or do we want to leave it up to the extending task to decide?

Reply:
That's a fair point. As we use it, we'd want the except schedule, but someone could want to use the migrate job on a schedule.

Reply:
I don't understand why you would ever run the migrate job on a schedule. Build, sure! But not migrate

Reply:
Sounds like we agree then - the element to be included doesn't need the except, that's up to the task that uses it.

Reply:
So don't add this then?